server/test/BUILD

```diff
@@ -29,7 +29,7 @@ package(default_visibility = ["//visibility:private",])
 host_compatible_java_test(
     name = "test-configuration",
     srcs = [
-        "CoreConfigTest.java",
+        "parameters/CoreConfigTest.java",
```

Comment:
we have relocated the test contents into a package that mirrors the source to be able to access package-private methods